bin.src/exportGraphInputs.py

```diff
@@ -18,29 +17,34 @@
     with(open(args.graph, "rb")) as f:
         graph = pickle.load(f)
 
-    # dataset_types_to_save = ("brightObjectMask", "ps1_pv3_3pi_20170110", "jointcal_photoCalib",
-    #                          "jointcal_wcs", "bias", "dark", "flat", "sky", "raw",
+    # dataset_types_to_save = ("brightObjectMask", "ps1_pv3_3pi_20170110",
+    #                          "jointcal_photoCalib", "jointcal_wcs", "bias",
+    #                          "dark", "flat", "sky", "raw",
     #                          "camera", "bfKernel")
     dataset_types_to_exclude = ("raw", "postISRCCD", "icExp", "icExpBackground", "icSrc")
 
-    with butler.export(directory=args.output, format="yaml", transfer="copy") as export:
+    with butler.export(directory=args.output, format="yaml", transfer="auto") as export:
         items = []
         for graph_node in graph:
             for quantum in graph_node.quanta:
                 for key, value in quantum.predictedInputs.items():
                     for dataset in value:
-                        if(dataset.datasetType.name not in dataset_types_to_exclude):
-                            items.append(dataset)
+                        if dataset.datasetType.name not in dataset_types_to_exclude:
+                            # The quantum graph doesn't know the ID of the
+                            # real dataset so convert to a real ref
+                            found = set(butler.registry.queryDatasets(dataset.datasetType.name,
+                                                                      collections=...,
+                                                                      dataId=dataset.dataId))
+                            items.extend(found)
         export.saveDatasets(items)
 
-    with butler.export(directory=args.output, filename="junk.yaml", format="yaml", transfer="copy") as export:
+    # This is solely to export the raw files. We do not need the yaml
```

Comment:
So why are we versioning it? Can't we remove it if it's not necessary, rather than committing it to the repository?